fixieai/agents/corpora.py

```diff
@@ -22,4 +151,6 @@ class DocumentCorpus:
     """A list of wildcard patterns to exclude from crawled URLs (e.g., */no_crawl/*)."""
 
     auth_token_func: Optional[str] = None
-    loader: Optional[DocumentLoader] = None
+
+
+DocumentCorpus = UrlDocumentCorpus  # Deprecated: Prefer UrlDocumentCorpus
```

Comment:
Is there a way to specify the new corpus type?

Reply:
Whoops, I forgot to add the registered function to the agent metadata. 😅 Added.